### PR TITLE
Add components and relationships count in model response

### PR DESCRIPTION
**Pull request by Yashsharma1911**
**Description**

This PR fixes #

This PR updates meshmodel API response to include component count and relationship count and implement feature for trim param (if true API will send only necessary data)

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Yashsharma1911** [comment]
This PR depends on new release of schemas repo

**Comment by Yashsharma1911** [comment]
lint will pass as we will make release of schemas repo

**Comment by leecalcote** [comment]
> lint will pass as we will make release of schemas repo

@Yashsharma1911 Ok. Let's do it. ✅

File: models/meshmodel/registry/v1beta1/model_filter.go [hunk 1]
```diff
@@ -182,27 +185,48 @@ func (mf *ModelFilter) Get(db *database.Handler) ([]entity.Entity, int64, int, e
...
+				}
+				_modelDB.Components = components
+			}
+			if includeRelationships {
+				var relationships []relationship.RelationshipDefinition
+				finder := db.Model(&relationship.RelationshipDefinition{}).
+					Select("relationship_definition_dbs.*").
+					Where("relationship_definition_dbs.model_id = ?", _modelDB.Id)
+				if err := finder.Scan(&relationships).Error; err != nil {
+					return nil, 0, 0, err
```
Comment (leecalcote) [inline models/meshmodel/registry/v1beta1/model_filter.go:224]: @Jougan-0 look good?

Comment (Jougan-0) [reply models/meshmodel/registry/v1beta1/model_filter.go:224]: Yes

**Review by Jougan-0 — APPROVED** [review]

**Pull request merged**